exporter/otlphttpexporter/otlp.go

```diff
@@ -270,7 +275,10 @@ func tracesPartialSuccessHandler(protoBytes []byte) error {
 	return nil
 }
 
-func metricsPartialSuccessHandler(protoBytes []byte) error {
+func metricsPartialSuccessHandler(protoBytes []byte, contentType string) error {
+	if contentType != protobufContentType {
+		return nil
```

Comment:
@atoulme Why ignore the response if it is not Protobuf? It can be OTLP/JSON which is a valid content type and which can be decoded, right?

Reply:
Nevermind, I see OTLP/JSON is not yet implemented by this exporter.